pages/find-locations/index.md

```diff
@@ -1,6 +1,11 @@
 ---
-title: Facility Locator
+title: Find VA Locations | Veterans Affairs
```

Comment:
The title tag is outputting like this -

```
<title>Find VA Locations | Veterans Affairs: VA.gov</title>
```
 
Per this [comment](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17409#issuecomment-480360100), it seems that the `: VA.gov` should replace the `:VA.gov`.